roles/openshift_prometheus/tasks/install_prometheus.yaml

```diff
@@ -0,0 +1,223 @@
+---
```

Comment:
@liggitt @sdodson @smarterclayton, I would love to have a conversation about this, specifically the topics that I'd like to see covered:

- Where the responsibility lies on managing the object(s)/technology being deployed?
- How do we achieve idempotency?
- What pattern is sustainable with idempotency in mind?